DQM/SiStripMonitorHardware/src/FEDHistograms.cc

```diff
@@ -369,6 +415,28 @@ void FEDHistograms::bookTopLevelHistograms(DQMStore::IBooker & ibooker , std::st
 		  "APV-ID",
 		  "FED-ID");
 
+  book2DHistogram( ibooker , fedErrorsVsId_,"FEDErrorsVsId",
```

Comment:
Please check that fedErrorsVsId_ is initialized. book2DHistogram has a nasty if statement...
I suspect in some cases it does not properly book the element and causes the segfault you are receiving.